client/ayon_ftrack/plugins/publish/integrate_ftrack_instances.py

```diff
@@ -530,11 +530,27 @@ def _get_asset_version_status_name(self, instance):
     def _prepare_component_metadata(
         self, instance, repre, component_path, is_review=None
     ):
+        """
+
```

Comment:
Should we put some more details in this docstring?
```suggestion
        """Return representation file metadata, like width, height, fps.

        This will only return any data for file formats matching a known
        video or image extension and may pass with only a warning if it
        was unable to retrieve the metadata from the image of video file.

```